aioredis/commands/server.py

```diff
@@ -112,9 +125,21 @@ def flushdb(self):
         fut = self._conn.execute('FLUSHDB')
         return wait_ok(fut)
 
-    def info(self, section):
-        """Get information and statistics about the server."""
-        # TODO: check section
+    def info(self, section='default'):
+        """Get information and statistics about the server.
+
+        If called without argument will return default set of sections.
+        For available sections, see http://redis.io/commands/INFO
+
+        :raises TypeError: if section is not string
+        :raises ValueError: if section is invalid
+
+        """
+        if not isinstance(section, str):
+            raise TypeError("section must be str")
+        section = section.lower()
+        if section not in self.INFO_ARGS:
+            raise ValueError("invalid section")
```

Comment:
I'm ok with default and docstring update,
but I don't see any reasoning for `INFO_SECTIONS` and `INFO_ARGS`:
first is used only in test and the second constraints user's input.
The later is not realy good idea because if new sections appeared aioredis won't let users to query them.